### ISSUE TITLE
[Issue] Symmetry to point doesn't work

### ISSUE THREAD

**Issue by YosypC** [realthunder/FreeCAD_assembly3#718]
Trying to put a point in the middle of an edge but the program wouldn't let me. I have tried selecting all the vertices under the spline on a different rectangle which had only the upper spline (so that I managed the fewest possible cases) but it still gives the same error. Am I doing something wrong?
file: 
[RotaryKnob.zip](https://github.com/realthunder/FreeCAD_assembly3/files/6787451/RotaryKnob.zip)

https://user-images.githubusercontent.com/50027752/124955167-23894380-e017-11eb-9bdc-195cbbb77485.mp4

edit: I tried using a line and this time worked...
edit2: still not working by setting a line between the two spline vertices.

edit3: tried different approach, still nothing. The vertical construction line 

https://user-images.githubusercontent.com/50027752/124962330-c72a2200-e01e-11eb-82b6-48de736a0102.mp4

OS: Windows 10 Version 2009
Word size of OS: 64-bit
Word size of FreeCAD: 64-bit
Version: 2021.614.24301 +3747 (Git)
Build type: Release
Branch: LinkDaily
Hash: ecbfba008c481db4cb17bf8bd514b2d12690a8e5
Python version: 3.8.6+
Qt version: 5.15.2
Coin version: 4.0.1
OCC version: 7.5.0
Locale: Italian/Italy (it_IT)

**Comment by luzpaz** [comment]
Please get in the habit of always copy/pasting your full [FC About](https://wiki.freecadweb.org/About) info

**Comment by Rabbit-sk** [comment]
Delete this constraint:
![obrázok](https://user-images.githubusercontent.com/74265320/125035181-c54d7680-e091-11eb-9fef-fc509ad5cb13.png)
here:
![obrázok](https://user-images.githubusercontent.com/74265320/125035448-1e1d0f00-e092-11eb-87f9-c68652fc6391.png)
and here:
![obrázok](https://user-images.githubusercontent.com/74265320/125035808-8c61d180-e092-11eb-9f10-97c46a907436.png)
before you apply the symmetry constraint.

In this case, the point on line constraint is redundant. Sketcher has sometimes problem to auto remove redundant constraints and needs some help from the user. 

This problem of not removing redundant constraints should be resolved, as the procedure I have advised you is non-intuitive, gained through practice, and this issue is a problem for the new user.

**Comment by YosypC** [comment]
> Please get in the habit of always copy/pasting your full [FC About](https://wiki.freecadweb.org/About) info
Understood, original updated.


> Delete this constraint:
> ![obrázok](https://user-images.githubusercontent.com/74265320/125035181-c54d7680-e091-11eb-9fef-fc509ad5cb13.png)

Didn't work, it tells me to delete other useful constraints, like dimensionals. Plus, when I click "Update" in the sketcher it gives me this error:
`Recompute failed! Please check report view.` 
Thanks for the support!

**Comment by Rabbit-sk** [comment]
> Didn't work

I created a new sketch containing only some geometry (point on line, two lines) and the procedure I advised you worked in it. In your case, I think the solver handled it incorrectly for some reason. Something similar has already happened to me with more complex sketches.

**Comment by herrgerd** [comment]
![point](https://user-images.githubusercontent.com/24494466/125208569-94d52a80-e293-11eb-9b60-af3b8e561b67.gif)
When using the symmetric constraint, the point can't be on the line before. Don't know why but it's always been like that.

Another example I often need:
![point_2](https://user-images.githubusercontent.com/24494466/125208661-25136f80-e294-11eb-909f-f1df6384d823.gif)

**Comment by YosypC** [comment]
> When using the symmetric constraint, the point can't be on the line before. Don't know why but it's always been like that.
Not true. I've always done the symmetry while the point sit on the line. However, doing the it the way you pointed out worked in my sketch. Besides, it doesn't make any sense; the point is free to move up and down to "look" for the center, it has no reason to be overconstrained.

I also noted that every time I place a Point, it prompts me the message: Recompute failed! Please check report view.

**Comment by herrgerd** [comment]
@YosypC @realthunder seems to work now. Thanks for fixing!

**Comment by YosypC** [comment]
It still does not work in my sketch.
